imu_sensor_notes.md

```diff
@@ -120,17 +120,17 @@ The SPI `gyro calibration`, includes 2 important values for each axis:
 
 The `cal_gyro_offset` is the offset when the Joy-Con is stable (held still).
 
-The `cal_gyro_coeff` is the coeff that is used in the equation and it's always `x343B` (`13371`).
+The `cal_gyro_coeff` is the coeff that is used in the equation.
```

Comment:
The `cal_gyro_coeff` is used for calibrating sensitivity. The default uncalibrated value for ±2000dps sensitivity is `x343B` (`13371`). Advise [here](spi_flash_notes.md#6-axis-sensor-factory-and-user-calibration).